input/data/2013/2013ApJ...764...38A/tev-000001.yaml

```diff
@@ -1,4 +1,25 @@
 source_id: 1
 reference_id: 2013ApJ...764...38A
 
-# TODO: add data
+data:
+  livetime: 41.25
+  significance: 7.5
+
+pos:
+  ra: {val: 0h6m26s, err: 0.09d, err_sys: 50s}
+  dec: {val: 72d59m01.0s, err: 0.04, err_sys: 50s}
+
+morph:
+  type: gauss
+
+  sigma: {val: 0.30, err: 0.03}
+  sigma2: {val: 0.24, err: 0.03}
+  # TODO: frame {radec, galactic} information is missing in the paper
```

Comment:
The position angle is in RADEC coordinates. I'm 99% sure because they quote position as RADEC and say "west of north" which would be very uncommon for GALACTIC.
So can you please update the TODO comment.
I would still leave it, because we haven't decided yet if we'll do "west of north" or some other orientation.
That "west of north" info should be mentionend in the comment.